src/backend/db/products.js

```diff
@@ -10,30 +10,192 @@ export const products = [
     _id: uuid(),
     imageUrl: "Images/kyotPeek.jpg",
     isLiked: false,
-    title: "kyotPeek",
-    color: "white",
-    price: "18000",
-    rating: "4.2",
+    title: "Exquisite Blanket",
+    color: "brown",
+    prevPrice:"18999",
+    price: 18000,
+    reviews: "3439",
+    rating: 4.6,
+    categoryName: "pattern-blankets",
+  },
+  {
+    _id: uuid(),
+    imageUrl: "Images/pawPlainBlanky.png",
```

Comment:
You can add the images here directly. No need to create extra folder for it.

Reply:
oh, alright, will see to it
thanks

Reply:
as for the images part, i have kept them in public thats why i am supplying that path in the db itself